README.md

```diff
@@ -1,4 +1,4 @@
-# NYC Mayor's Office of the CTO Landing Page as a Service 2.0
+# NYC Mayor's Office of the Chief Technology Officer Landing Page as a Service 2.0
```

Comment:
This seems very long. Maybe "NYC Mayor's Office of the CTO LPaaS 2.0" or "NYC Mayor's Office of the CTO Landing Page as a Service 2.0"?

Reply:
agreed - changed to "NYC MOCTO Landing Page as a Service 2.0"